### PR TITLE
feat(syndicated): adding new scenario for syndicated page layout

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

File: src/pages/solve-overview/index.js [hunk 1]
```diff
@@ -76,16 +77,27 @@ export default Vue.component('solve-overview-content', {
...
       if (typeof topic !== 'undefined') {
-        const cats = await this.getCategories();
-        const catData = _.find(cats, c => c.tid === topic);
-        if (catData) {
-          this.topic.header = catData.name;
-          this.topic.desc = catData.description;
+        // seperate syndicated and tid categories here
+        if (this.isSyndicatedPage) {
+          this.topic.header = 'Mike\'s Unprepaired Syndicated Title';
+          this.topic.desc = 'Lorem Ipsum';
```
Comment (WidgetsBurritos) [inline src/pages/solve-overview/index.js]: Oh actually, until we get the follow-up ticket in to use taxonomy here, we need to force this text through the translation filter.

Comment (runrog) [reply src/pages/solve-overview/index.js]: This has been updated

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5570-syndicated-layout.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

**Pull request merged**